### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by repasics**
Fixed binary operator generator to work with python 2.7.

### CODE REVIEW

**Review by rerobika — COMMENTED** [review]
Could you please provide a bit more detailed description about what exactly happened here?

File: src/scripts/generate_binary_ops.py [hunk 1]
```diff
@@ -137,24 +137,29 @@ def sub(self, left_value, right_value):
...
         return self.create_operation_result_object('*', right_value, result)
 
     # Division (floor division)
     def div(self, left_value, right_value):
         if right_value == 0: # To avoid division by zero
-            if (random.randint(0, 1)):
-                right_value = random.randint(1, _constants.max)
-            else:
-                right_value = random.randint(_constants.min, - 1)
+            while True:
```
Comment (loki04) [inline src/scripts/generate_binary_ops.py]: I do not like the `while True` loops if it depends on random number. Consider to use something like this:
```
right_value = random.randint(_constants.min, _constants.max)
if right_value == 0:
  right_value = random.randint(1, _constants.max)
```

**Review by loki04 — CHANGES_REQUESTED** [review]

**Comment by repasics** [comment]
At multiplication the random number's range sometime started with higher number than it's end value. The python 2.7 could not handle it so I granted the start number be always lower than the end value.
The divisions result +=  1 if it is lower than zero is because rounding.

File: src/scripts/generate_binary_ops.py [hunk 2]
```diff
@@ -138,24 +138,27 @@ def sub(self, left_value, right_value):
...
-        if right_value == 0: # To avoid division by zero
-            if (random.randint(0, 1)):
-                right_value = random.randint(1, _constants.max)
-            else:
-                right_value = random.randint(_constants.min, - 1)
+        right_value = random.randint(_constants.min, _constants.max)
+        if right_value == 0:
+            right_value = random.randint(1, _constants.max)
         result = left_value / right_value
-        result = int(result) # Rounding because it might not be an integer
```
Comment (loki04) [inline src/scripts/generate_binary_ops.py:158]: If you are about to fix 2.7 please check both 2.7 and 3.x as well. The current version generates different values!
Consider the differences of the division operator `/` between 2.7 and 3.x!

Lets see if the random returns 9999
3.x:
In [39]: div(-9998,0)
Out[39]: ['/', 9999, 0.00010001000100012813]
In [40]: div(-9999,0)
Out[40]: ['/', 9999, 0.0]
In [41]: div(-10000,0)
Out[41]: ['/', 9999, -0.00010001000100001711]

2.7:
In [5]: div(-9998,0)
Out[5]: ['/', 9999, 0]
In [6]: div(-9999,0)
Out[6]: ['/', 9999, 0]
In [7]: div(-10000,0)
Out[7]: ['/', 9999, -1]

**Review by loki04 — CHANGES_REQUESTED** [review]

File: src/scripts/generate_binary_ops.py (hunk 2 above)
Comment (repasics) [reply src/scripts/generate_binary_ops.py:158]: I fixed it. Both 2.7 and 3.6 generates the right value now

**Review by loki04 — APPROVED** [review]
LGTM

**Review by rerobika — APPROVED** [review]
LGTM

**Pull request merged**